sale_order_digitized_signature/report/report_saleorder.xml

```diff
@@ -1,12 +1,15 @@
-<?xml version="1.0" ?>
 <odoo>
 
-    <template id="report_saleorder_document" inherit_id="sale.report_saleorder_document" priority="9999">
+    <template id="report_saleorder_document"
+              inherit_id="sale.report_saleorder_document"
+              priority="9999">
```

Comment:
The priority is useless as we are not doing any replace

Reply:
Ping @mgosai